vmdb/app/models/miq_enterprise.rb

```diff
@@ -57,6 +66,38 @@ def my_zone
     MiqServer.my_zone
   end
 
+  def miq_regions
+    MiqRegion.all
+  end
+
+  def ext_management_systems
+    ExtManagementSystem.all
+  end
+
+  def vms_and_templates
+    VmOrTemplate.where("ems_id IS NOT NULL")
```

Comment:
Prefer newer syntax in the lambda you deleted:

`VmOrTemplate.where.not(:ems_id => nil)`